Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 
 gem 'rake'
 gem 'mumukit', github: 'mumuki/mumukit', tag: 'v0.3.0'
+gem 'gobstones-spec', github: 'uqbar-project/gobstones-spec', branch: 'transform-code-to-gem'
```

Comment:
This is just for testing purposes, tag v0.1.0 should be used instead (after merging https://github.com/uqbar-project/gobstones-spec/pull/1)